consult.el

```diff
@@ -1410,10 +1423,10 @@ The function can be used as the `:state' argument of `consult--read'."
                 (set-buffer saved-buffer)
                 (narrow-to-region saved-min saved-max)
                 (goto-char saved-pos)))
-            ;; Handle positions with overlay information
-            (consult--jump-1 (or (car-safe cand) cand))
-            (setq invisible (consult--invisible-open-temporarily)
-                  overlays
+          ;; Handle positions with overlay information
+          (consult--with-jump (or (car-safe cand) cand)
+            (setq consult--preview-previous-invisibles (consult--invisible-open-temporarily)
+                  consult--preview-previous-overlays
```

Comment:
I feel `invisible` should be defined for not local but global because `void variable` error happened.

Reply:
Yes, this is wrong. Please revert the change to the overlays variable.